watertap/unit_models/tests/test_electrodialysis_0D.py

```diff
@@ -687,7 +687,7 @@ def test_initialization_scaling(self, electrodialysis_cell3):
             "flow_mol_phase_comp", 1e4, index=("Liq", "Cl_-")
         )
         m.fs.properties.set_default_scaling(
-            "flow_mol_phase_comp", 1e5, index=("Liq", "N")
+            "flow_mol_phase_comp", 1e6, index=("Liq", "N")
```

Comment:
I included `IonExchangeType()` in the unit model itself in anticipation of using it in future refinements of model. I'd prefer to keep it in IX for that reason. 

I didn't make the second change, maybe it is a result of me merging from main? Doesn't matter if it is reverted or not.

Reply:
This is a change made by me (I wrote the corresponding file) and it is a trivial change due to a previous failed check but things have been updated since then.  Mark this resolved if no new error is related to this.